packages/typescript/src/index.ts

```diff
@@ -125,11 +125,12 @@ export default function typescript(options: RollupTypescriptOptions = {}): Plugi
         const output = findTypescriptOutput(ts, parsedOptions, fileName, emittedFiles, tsCache);
         output.declarations.forEach((id) => {
           const code = getEmittedFile(id, emittedFiles, tsCache);
-          if (!code) return;
+          const baseDir = outputOptions.dir || (tsconfig && tsconfig.substring(0, tsconfig.lastIndexOf('/')));
```

Comment:
I think this is a little hard to read, I'd rather just use a few if statements.
```suggestion
          let baseDir = outputOptions.dir;
          if (!baseDir && tsconfig) {
            baseDir = tsconfig.substring(0, tsconfig.lastIndexOf('/'));
          }
```

Reply:
done. thanks for the quick review!